src/main/java/controller/AttendanceController.java

```diff
@@ -0,0 +1,176 @@
+package controller;
+
+import domain.AbsentPolicy;
+import domain.AttendanceDateTime;
+import domain.AttendanceSheet;
+import domain.AttendanceSheets;
+import domain.AttendanceSheetsFactory;
+import domain.AttendanceState;
+import util.FileReaderUtil;
+import view.InputView;
+import view.OutputView;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AttendanceController {
+
+    private final InputView inputView;
+
+    public AttendanceController(InputView inputView) {
+        this.inputView = inputView;
+    }
+
+    public void run() {
+        LocalDate date = LocalDate.of(2024, 12, 13);
+
+        AttendanceSheetsFactory attendanceSheetsFactory = new AttendanceSheetsFactory(new FileReaderUtil());
+        AttendanceSheets attendanceSheets = attendanceSheetsFactory.create();
+
+        while (true) {
+            String select = inputView.inputMenu(LocalDate.now());
```

Comment:
![image](https://github.com/user-attachments/assets/9e7421e0-1712-4ae5-a04c-588562df55e8)
```suggestion
            String select = inputView.inputMenu(LocalDate.of(2024, 12, 13));
```
오늘 날짜도 2024.12.13으로 가정해보면 어떨까요~?
또 프로그램 정책상 예외가 발생하더라도 프로그램이 종료되면 안될텐데 확인 부탁드려요 😄 
`프로그램은 사용자가 종료할 때까지 종료되지 않으며, 해당 기능을 수행한 후 초기 화면으로 돌아간다.`